src/app/head/hooks/locationChange.ts

```diff
@@ -2,6 +2,10 @@ import { locationChange } from '../../navigation/actions';
 import { ActionHookBody } from '../../types';
 import { actionHook, assertDefined } from '../../utils';
 
+if (typeof(document) !== 'undefined') {
+  import(/* webpackChunkName: "NodeList.forEach" */ 'mdn-polyfills/NodeList.prototype.forEach');
+}
```

Comment:
ditto this re comment above

Reply:
this one is duplicated from Page.tsx, but i like to do a thing where i include the polyfill in each place that its used, so that:
- if the code is split its always included when its needed
- its close to the code that its needed for so if you update the code to need or not need polyfills you can change them right there

that said i started a pattern on #315 that might be more clear and save 4 lines of code